iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -91,6 +90,7 @@ Inspector.prototype.handleMessage = function (event) {
     var data = JSON.parse(jsonString);
 
     if (data.task == 'SHOW_FACT') {
+        this.showDocumentForFactId(data.factId);
         this.selectFact(data.factId);
```

Comment:
Does `handleFactDeepLink` need to be similarly updated?  Perhaps both should be using a common function (`showAndSelectFact`?) and the `showDocumentForFactId` logic should be there?

Reply:
Actually, this was a bug introduced by merging in the refactoring done for ix:continuation (c55557c).  selectFact already switches tabs if needed.  Fixed in c3ad9f5.